sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -883,21 +902,20 @@ def _is_in_terminal_state(self):
     if not self.has_job:
       return True
 
-    return self.state in [
-        PipelineState.STOPPED, PipelineState.DONE, PipelineState.FAILED,
-        PipelineState.CANCELLED, PipelineState.DRAINED]
+    values_enum = dataflow_api.Job.CurrentStateValueValuesEnum
+    return self._job.currentState in [
```

Comment:
<!--new_thread; commit:5b7323ad9e10a5693ea41a239c8a675139b977de; resolved:0-->
Do we want to factor this list out somewhere (or add a TODO to do so)?

Reply:
Added a TODO.